### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by tullsta2** [nextcloud/all-in-one#4745] [Questions] [answered]
Hello,

my AIO instance runs now for about 18 months quite smoothly.
Without any changes/warning I today encountered an "Internal Server Error":

> Internal Server Error
> The server was unable to complete your request.
> 
> If this happens again, please send the technical details below to the server administrator.
> 
> More details can be found in the server log.
> Technical details
> 
>    Remote Address: xxx.xxx.xxx.xxx
>    Request ID: 

AIO v8.2.1 runs in a docker environment under openmediavault v6.9.16 (Linux 6.1.0-0.deb11.13-amd64) and (at least to my knowledge) IPv6 is not enabled.
When scanning through the docker log-files I could not find any error message.

I can log into the AIO interface and tried to shotdown all containers and restart (from AIO interface). Container status is green.

It would be nice if someone could help give me a hint where to look further / how to debug this issue.

One thing that puzzled me though is that the "nextcloud.log"-file in the nc data directory is empty and it belongs to group owner is "root"where I would have expected "www-data".

Thanks for any support - tullsta

**Comment by szaimen** [comment, marked as answer]
Hi, you should be able to retreive the log with `sudo docker exec -it nextcloud-aio-nextcloud tail data/nextcloud.log`

**Reply by tullsta2** [reply]
Hello Simon, thanks for the support. Here the report for the last reqID.
To be honest: I would now not know how t provcees from here...

thanks + LG - tullsta

{
   "reqId":"myPnaRKX2qJNXROktIpL",
   "level":1,
   "time":"2024-06-03T14:48:08+00:00",
   "remoteAddr":"xxx.xxx.xxx.xxx",
   "user":"myUserName",
   "app":"no app in context",
   "method":"GET",
   "url":"/apps/dashboard/",
   "message":"Unable to generate a URL for the named route \"tasks.page.index\" as such route does not exist.",
   "userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0",
   "version":"28.0.5.1",
   "exception":{
      "Exception":"Symfony\\Component\\Routing\\Exception\\RouteNotFoundException",
      "Message":"Unable to generate a URL for the named route \"tasks.page.index\" as such route does not exist.",
      "Code":0,
      "Trace":[
         {
            "file":"/var/www/html/lib/private/Route/Router.php",
            "line":380,
            "function":"generate",
            "class":"Symfony\\Component\\Routing\\Generator\\UrlGenerator",
            "type":"->",
            "args":[
               "tasks.page.index",
               [
               ],
               1
            ]
         },
         {
            "file":"/var/www/html/lib/private/Route/CachingRouter.php",
            "line":65,
            "function":"generate",
            "class":"OC\\Route\\Router",
            "type":"->",
            "args":[
               "tasks.page.index",
               [
                  
               ],
               false
            ]
         },
         {
            "file":"/var/www/html/lib/private/URLGenerator.php",
            "line":103,
            "function":"generate",
            "class":"OC\\Route\\CachingRouter",
            "type":"->",
            "args":[
               "tasks.page.index",
               [
                  
               ]
            ]
         },
         {
            "file":"/var/www/html/lib/private/NavigationManager.php",
            "line":374,
            "function":"linkToRoute",
            "class":"OC\\URLGenerator",
            "type":"->",
            "args":[
               "tasks.page.index"
            ]
         },
         {
            "file":"/var/www/html/lib/private/NavigationManager.php",
            "line":131,
            "function":"init",
            "class":"OC\\NavigationManager",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/apps/theming/lib/ThemingDefaults.php",
            "line":199,
            "function":"getAll",
            "class":"OC\\NavigationManager",
            "type":"->",
            "args":[
               "guest"
            ]
         },
         {
            "file":"/var/www/html/lib/private/legacy/OC_Defaults.php",
            "line":271,
            "function":"getShortFooter",
            "class":"OCA\\Theming\\ThemingDefaults",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/lib/public/Defaults.php",
            "line":176,
            "function":"getLongFooter",
            "class":"OC_Defaults",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/core/templates/layout.guest.php",
            "line":51,
            "function":"getLongFooter",
            "class":"OCP\\Defaults",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/lib/private/Template/Base.php",
            "line":180,
            "args":[
               "/var/www/html/core/templates/layout.guest.php"
            ],
            "function":"include"
         },
         {
            "file":"/var/www/html/lib/private/Template/Base.php",
            "line":150,
            "function":"load",
            "class":"OC\\Template\\Base",
            "type":"->",
            "args":[
               "/var/www/html/core/templates/layout.guest.php",
               null
            ]
         },
         {
            "file":"/var/www/html/lib/private/legacy/OC_Template.php",
            "line":139,
            "function":"fetchPage",
            "class":"OC\\Template\\Base",
            "type":"->",
            "args":[
               null
            ]
         },
         {
            "file":"/var/www/html/lib/private/legacy/OC_Template.php",
            "line":170,
            "function":"fetchPage",
            "class":"OC_Template",
            "type":"->",
            "args":[
               null
            ]
         },
         {
            "file":"/var/www/html/lib/private/Template/Base.php",
            "line":132,
            "function":"fetchPage",
            "class":"OC_Template",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/lib/private/legacy/OC_Template.php",
            "line":291,
            "function":"printPage",
            "class":"OC\\Template\\Base",
            "type":"->",
            "args":[
               
            ]
         },
         {
            "file":"/var/www/html/index.php",
            "line":121,
            "function":"printExceptionErrorPage",
            "class":"OC_Template",
            "type":"::",
            "args":[
               [
                  "Error"
               ],
               500
            ]
         }
      ],
      "File":"/var/www/html/3rdparty/symfony/routing/Generator/UrlGenerator.php",
      "Line":144,
      "message":"Unable to generate a URL for the named route \"tasks.page.index\" as such route does not exist.",
      "exception":{
         
      },
      "CustomMessage":"Unable to generate a URL for the named route \"tasks.page.index\" as such route does not exist."
   }
}

**Comment by tullsta2** [comment]
Hello all,

thanks Simon for the help! 
With the information from another post [here](https://help.nextcloud.com/t/internal-server-error-after-docker-compose-update/194168) I could confirm that the issue was created by the circles-app (indeed the desktop-app was hinting towards that app).
Disabling circles via occ did the job.

Rem: could also be that bookmarks-app was the cause (I'm also using this one): this one got a hot fix to 14.1.1 on the same day as releasing 14.1.0:
14.1.1 Fixed => Fix circle usage in code to avoid dependency hell

Thanks + kind regards + LG - tullsta
